rllib/BUILD

```diff
@@ -3182,6 +3182,15 @@ py_test(
     args = ["--as-test", "--framework=torch", "--stop-reward=70.0", "--num-cpus=4"]
 )
 
+py_test(
+    name = "examples/multi_agent_cartpole_torch_w_rlm",
```

Comment:
"rlm" -> "rl_module"

Also: see my comment below: Can we use a command line arg here (`--use-rl-module`)?

Reply:
answered below.